src/components/button/index.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import Button from './index.tsx';
+
+<Meta title="Components/Button" component={Button} />
+
+export const Template = (args) => <Button {...args}>Primary button</Button>;
+
+# Button
+
+## Overview
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      isDisabled: false,
+    }}
+    argTypes={{
+      variant: {
+        options: ['primary'],
+        control: { type: 'radio' },
+        defaultValue: 'primary',
+      },
+      size: {
+        options: ['md', 'lg', 'xl'],
+        control: { type: 'radio' },
+        defaultValue: 'lg',
+      },
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Sizes
+
+<Canvas>
+  <Story name="Sizes">
+    <Button size="md">Medium</Button>
+    <Button size="lg">Large</Button>
+    <Button size="lg">Large</Button>
```

Comment:
```suggestion
    <Button size="xl">Large</Button>
```

Maybe we can use Dummy text on the buttons OR we can use the `robot` size naming to do not confused people with the name of the buttons here on the code and on figma. What do you think?

Reply:
Sorry for the confusion, I some miss added the `xl` button size.